includes/payments/class-edd-payment.php

```diff
@@ -3160,6 +3160,12 @@ private function setup_user_info() {
 			);
 		}
 
+		// Check for old `user_info` meta which may still exist.
+		$old_meta = $this->get_meta( 'payment_meta' );
```

Comment:
I think we should be able to just use `edd_get_order_meta()` here. That's preferable because it will bypass some of the old filters and backwards compat stuff that's in `$this->get_meta()`. I don't think we need all that here.